README.md

```diff
@@ -34,13 +34,12 @@ A bit more about the different fields...
 - `allow_private` (optional)
     - bool [true, false]
     - default: false
-    - If false bugs that are not public will not be synchronized. Note that in order to synchronize
-      private bugs the bugzilla user that JBI runs as must be in the security groups that are making
-      the bug private.
-- `bugzilla_user_id`
-    - a bugzilla user id, a list of user ids, or a literal "tbd" to signify that no bugzilla user id is available
-    - If an issue arises with the workflow, communication will be established with these users
-    - Please enter the user information for one or more stakeholders
+    - If `false`, bugs will not be synchronized if they are not made public. Note that in order to synchronize private bugs,
+     the bugzilla user that JBI runs as must be in the security groups that are making the bug private.
+- `contact`
+    - an email address, a list of email addresses, or a literal "tbd" to signify that no contact is available
+    - If an issue arises with the workflow, communication will be established with these contacts
+    - Please enter the contact information for one or more stakeholders
```

Comment:
I think there was an issue with the merge of changes made in #440 

I'm sorry but you would have to adjust the changes one more time :(

Reply:
I have made modifications according to the new commits. Hope its fine now. Let me know if you want more changes.